mk_verificator/tests/rabbit_mq/test_rabbit_cluster.py

```diff
@@ -1 +1,33 @@
-__author__ = 'msenin'
+__author__ = 'pivanets'
+import json
+
+
+def test_checking_rabbitmq_cluster(local_salt_client):
+    cs = local_salt_client.cmd(
```

Comment:
and, please, use` salt-call rabbitmq.cluster_status` for gathering **actual result**. 
`salt-call pillar.data --output json rabbitmq:cluster` shows you salt settings, in other words that's only **expected result**